pkg/storage/ec/client.go

```diff
@@ -267,8 +260,8 @@ func (ec *ecClient) Repair(ctx context.Context, limits []*pb.AddressedOrderLimit
 		}
 	}()
 
-	if successfulCount < int32(optimalCount) {
-		return nil, nil, Error.New("successful nodes count (%d) does not match optimal count (%d) of erasure scheme", successfulCount, optimalCount)
+	if successfulCount == 0 {
+		return nil, nil, Error.New("successful nodes count %d", successfulCount)
```

Comment:
As `successfulCount` is 0 here for sure, perhaps we can have a clearer error message like "repair to all nodes failed".